package.json

```diff
@@ -33,7 +33,7 @@
         "@types/sinon": "~7.0.9",
         "chai": "~4.2.0",
         "chai-spies": "~1.0.0",
-        "coveralls": "~3.0.2",
+        "coveralls": "^3.0.6",
```

Comment:
I think I always mix the semantics of `~` and `^`, but is this change an intended fix or a deviation from our *explicit version* policy?

Reply:
I did not change the file. I guess it was vs-code. Shall I revert the changes?

Reply:
@cgcostume What is the desired approach?